WORKSPACE

```diff
@@ -4,6 +4,11 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
     name = "com_google_absl",
+    # On MSVC's STL implementation, string_view cannot be constructed from
+    # a string_view::iterator. This patch forces the use of absl's string_view
```

Comment:
Is this a bug in the MSVC STL implementation that we should file with Microsoft, or is this within the frame of interpretation of the standard ?
Do you have an example of an error message this produces ?

Reply:
That appears to be a deliberate strict interpretation of the standard. I found [this Reddit](https://www.reddit.com/r/cpp/comments/bg7a52/stdstring_view_has_no_constructor_taking_its_own/) about it.

It can be fixed in the code by using replacing `.begin()` with `.data()` and `.end()` with `.data() + .length()`, but since it is used at several places in Verible's codebase, I think it is simpler to use absl's implementation, for now at least.

To be fair, I am not a programmer, especially if an OS is involved, so likely someone more used to `string_view` can find an easier fix.